README.md

```diff
@@ -1,60 +1,78 @@
+
+To deploy the demos, request the **Change Data Capture with Debezium** workshop from the [Red Hat Product Demo System (RHPDS)](https://rhpds.redhat.com).
+
+### Provisioning time
+
+Workshop provisioning takes 60 - 75 minutes. 
+
+If the workshop fails to deploy, resubmit your provisioning request. If deployment continues to fail after three attempts, open a ticket by sending a message to rhds-help@redhat.com.
 
-This workshop should be available after 60-75 minutes. It could fortuitously fail in the deployment, please just try again as it could be fixed already. If take more than 3 attempts, please do send an email to raise a ticket to: rhds-help@redhat.com
-Relevant information is sent in the final email
+After your provisioning request is received, the _Red Hat Product Demo System_ sends you a series of email messages with updates about the status of the request. When your deployment is ready for use, you receive a final email that includes information about how to access the environment. 
 
 ## Scenarios
 
-### Introducing Debezium
+### Introduction to Debezium
 
-In this scenario you are capturing events from a mysql database to Apache Kafka. You can go over the data structure of the event for each of the possible changes in the database: create, update and delete. 
+In this introductory scenario, you use Debezium to capture data change events from a MySQL database and send change event records to Apache Kafka. You can go over the data structure for each type of event record: creates, updates, and deletes. 
 
-This is a basic demo where everything is already set up for you to go over the solution pattern from the solution explorer web application.
+This is a basic demo where everything is already set up for you to go over the solution pattern from the Solution Explorer web application.
 
 #### Video
 
-### Data Replication
+### Data replication
 
-The second scenario covers how to avoid dual writes. You have a legacy PHP application that needs to update an Elasticsearch index with the new orders captured for future search. The CDC part is covered with Debezium connectors and includes the use of Single Message Transformations The second part consists of the deployment of the new Camel Kafka Connectors.
+The data replication scenario demonstrates how to use Debezium to avoid dual writes. In the scenario, you have a legacy enterprise application built on PHP, that sends orders and records into a database. To enable faster querying of your orders, you want to  send the records for each new order event to an Elasticsearch index. 
 
-This is a basic demo where everything is already set up for you to go over the solution pattern from the solution explorer web application
+The scenario makes use of Debezium connectors and Single Message Transformations (SMTs). The second part of the demo introduces the new Camel Kafka Connectors.
+
+The components used in this scenario are deployed in the Solution Explorer, and are ready for you to use.
```

Comment:
The components are deployed into the debezum-complete-demo namespace and the script to follow is in Solution Explorer.